website/blog/2024-01-10-roadmap.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: 2024-team-roadmap
+title: New year, new MACI
+authors:
+  name: Sam Richards
+  title: MACI team
+  url: https://x.com/samonchain
+  image_url: https://avatars.githubusercontent.com/u/8097623?v=4
+tags: [maci, roadmap]
+---
+
+Greetings anons,
+
+Happy 2024 and welcome to the new MACI documentation website and blog! Moving forward, we’ll post here with our latest project news and development updates.
+
+We have a few exciting announcements to share, but first a quick review:
+
+## MACI Overview
+
+### WTF is MACI?
+
+**Minimal Anti-Collusion Infrastructure (MACI) is an on-chain voting platform which protects privacy and minimizes the risk of collusion and bribery.**
+
+MACI is a set of smart contracts and zero-knowledge circuits upon which which developers can build applications, such as voting systems or quadratic funding platforms. It was originally [proposed by Vitalik Buterin](https://ethresear.ch/t/minimal-anti-collusion-infrastructure/5413) in 2019, then implemented and maintained by community members with support from the [Ethereum Foundation](https://ethereum.foundation/). It's now actively maintained within [Privacy & Scaling Explorations (PSE)](https://pse.dev/).
+
+For a general overview, brief history, and context on the importance of MACI, see the [Release Announcement: MACI 1.0](/blog/maci-1-0-release) by Wei Jie, one of the creators. He also created a helpful [overview of MACI video](https://www.youtube.com/watch?v=sKuNj_IQVYI). Kyle Charbonnet wrote a great [Technical Introduction to MACI 1.0](/blog/maci-1-0-technical-introduction) that provides a walkthrough on how MACI operates. I also recently presented an [introduction to MACI](https://app.streameth.org/zuzalu/zuconnect_hackathon/session/maci_workshop) at Devconnect.
+
+### MACI Vision
+
+MACI is a public good that is quickly becoming a core piece of infrastructure for Ethereum-based applications to support on-chain voting while protecting user privacy. Using MACI, no voter can prove how they voted yet voting results are published publicly and verified with cryptographic proofs to prevent censorship, bribery, collusion, and other nefarious acts common in public polling processes.
+
+**Our MACI team’s vision is to build the most secure e-voting solution in the world.**
+
+Let's hop into our updates to see how we plan to execute on this bold vision.
+
+## Project updates
+
+New year, new team, new roadmap!
+
+### 1) New team
+
+Over the past few months we've undergone lot of change within the MACI core team. I'd like to take this chance to introduce and welcome our new core members!
+
+MACI is now maintained and will be continuously improved by our core team:
```

Comment:
@ctrlc03 @0xmad @kittybest @crisgarner @aguzmant103 Vee - feel free to update (or tell me) which URL (GH, Twitter, personal website, etc.) you'd like you use here

Reply:
LGTM! I think if everyone uses GitHub, it makes sense to keep all the same